espnet2/tasks/enh.py

```diff
@@ -99,8 +102,11 @@
         sdr=SDRLoss,
         ci_sdr=CISDRLoss,
         si_snr=SISNRLoss,
+        coh=FrequencyDomainAbsCoherence,
         mse=FrequencyDomainMSE,
```

Comment:
I was thinking about to use mse_f and l1_f to be consistent with mse_t and l1_t. But it might break the pretrained models. I think it is fine.

Reply:
Yes, it is better to use `mse_fd` and `mse_td`. I can also keep the old keys for backward compatibility.